src/Scripting/Core/Hosting/Resolvers/RuntimeMetadataReferenceResolver.cs

```diff
@@ -171,7 +171,7 @@ private PortableExecutableReference ResolveTrustedPlatformAssemblyCore(string na
 
             if (_lazyTrustedPlatformAssemblies.TryGetValue(name, out string path) && File.Exists(path))
             {
-                return MetadataReference.CreateFromFile(path, properties);
+                return _fileReferenceProvider(path, properties);
```

Comment:
>return _fileReferenceProvider(path, properties); [](start = 16, length = 48)

Is this change necessary to fix the problem at hand?

Reply:
No. But I kept it for consistency with the other functions that create metadata references in this class.

---
In reply to: [322469608](https://github.com/dotnet/roslyn/pull/38550#discussion_r322469608) [](ancestors = 322469608)

Reply:
> No. But I kept it for consistency with the other functions that create metadata references in this class.

Please back this change by a unit-test that would fail without the change.

---
In reply to: [322495905](https://github.com/dotnet/roslyn/pull/38550#discussion_r322495905) [](ancestors = 322495905,322469608)